main.go

```diff
@@ -450,6 +425,18 @@ func abigen(solFile, contractName string, workDir string) (*cacheObject, error)
 	if err != nil {
 		return nil, xerrors.Errorf("solc: %w", err)
 	}
+	return compiled, nil
+}
+
+// Return the filename with its filename extension trimmed away.
+func trimExtension(filename string) string {
+	return strings.TrimSuffix(filename, path.Ext(filename))
+}
+
+// parseJsonBytecode takes compiled bytes, as the bytestream output by `solc --combined-json abi,bin,userdoc,devdoc`
```

Comment:
```suggestion
// parseJsonBytecode takes compiled bytes, as the bytestream output by
```

Reply:
Should we try to stick to 99 characters per line? We don't have to but I think we should